k4MarlinWrapper/src/components/TrackingCellIDEncodingSvc.cpp

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2019-2023 Key4hep-Project.
+ *
+ * This file is part of Key4hep.
+ * See https://key4hep.github.io/key4hep-doc/ for further info.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "k4MarlinWrapper/TrackingCellIDEncodingSvc.h"
+
+// k4fwcore
+#include <k4Interface/IGeoSvc.h>
+
+#include <GaudiKernel/MsgStream.h>
+
+DECLARE_COMPONENT(TrackingCellIDEncodingSvc);
+
+TrackingCellIDEncodingSvc::TrackingCellIDEncodingSvc(const std::string& name, ISvcLocator* svc)
+    : base_class(name, svc), m_geoSvc(svc->service("GeoSvc")) {}
```

Comment:
@hegner Is there a way to make the name of the "GeoSvc" configurable? Or directly give the geoSvc configured in python to this service?

Reply:
You can only put a string parameter, which you then use inside this code. A direct assignment of services to properties is unfortunately not there in current Gaudi.

Reply:
Ok, I will add a GeoSvcName property or something.
Thanks!